src/opentimelineio/serializableCollection.h

```diff
@@ -5,6 +5,7 @@
 
 #include "opentimelineio/composition.h"
 #include "opentimelineio/serializableObjectWithMetadata.h"
+#include "opentimelineio/timeline.h"
```

Comment:
This might be a good time to break out test_serializableCollection.cpp

Reply:
Good idea, I also broke out test_track.cpp and test_timeline.cpp since they also have ```children_if()```.

Reply:
awesome, thanks!